### PR TITLE
add dictionary management interface

### PR DESCRIPTION
**Pull request by downdawn**

### CODE REVIEW

**Comment by downdawn** [comment]
please review.

File: backend/app/models/sys_dict_data.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    __tablename__ = 'sys_dict_data'
+
+    id: Mapped[id_key] = mapped_column(init=False)
+    label: Mapped[str] = mapped_column(String(32), unique=True, comment='字典标签')
+    value: Mapped[str] = mapped_column(String(32), unique=True, comment='字典值')
+    type_id: Mapped[int] = mapped_column(ForeignKey('sys_dict_type.id'), comment='字典类型id')
+    sort: Mapped[int] = mapped_column(default=0, comment='排序')
+    status: Mapped[bool] = mapped_column(default=True, comment='状态（0停用 1正常）')
+    remark: Mapped[str | None] = mapped_column(LONGTEXT, default=None, comment='备注')
+    # 字典类型一对多
```
Comment (wu-clan) [inline backend/app/models/sys_dict_data.py:22]: I think the notes should be consistent：`# 字典数据一对多`

**Review by wu-clan — APPROVED** [review]

File: backend/app/models/sys_dict_data.py (hunk 1 above)
Comment (downdawn) [reply backend/app/models/sys_dict_data.py:22]: Ok

**Comment by wu-clan** [comment]
Sorry, my earlier comment was wrong 🤣

I modified and submitted it

**Pull request merged**